utils/utils.h

```diff
@@ -413,6 +413,12 @@ void stacktrace(void);
 		raise(SIGTRAP);                                                                    \
 	}
 
+#if defined(__x86_64__) || defined(__i386__)
+#define TRAP() asm volatile("int $3")
+#else
+#define TRAP() raise(SIGTRAP)
+#endif
```

Comment:
Maybe `ASSERT` and `DASSERT` can use it?

Reply:
Yes, right. I've just updated it.